test/reload.ps1

```diff
@@ -10,6 +17,9 @@ foreach ($module in $modules) {
     }
 }
 
-Register-SecretVault (Join-Path $PSScriptRoot '..' 'SecretManagement.LastPass.psd1') -Name 'LastPass.Tests'
```

Comment:
This didn't work with 5.1

Reply:
Yeah because in 5.1 `Join-Path` only took in 2 paths...

5.1 ... so limiting...